HT2/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "HT2",
+  "version": "1.0.0",
+  "description": "",
+  "main": "./dist/server.js",
+  "scripts": {
+    "start": "nodemon -e ts --exec \"npm run compile\"",
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "lint": "eslint . --ext .ts",
+    "compile": "tsc && node dist/server"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
+    "@types/express": "^4.17.11",
+    "@types/node": "^14.14.35",
+    "@typescript-eslint/eslint-plugin": "^4.19.0",
+    "@typescript-eslint/parser": "^4.19.0",
+    "eslint": "^7.22.0",
+    "ts-node": "^9.1.1",
+    "typescript": "^4.2.3"
+  },
+  "dependencies": {
+    "express": "^4.17.1",
+    "express-joi-validation": "^5.0.0",
+    "joi": "^17.4.0",
```

Comment:
extra comma resulting in bad json 
<img width="1106" alt="Screen Shot 2021-04-08 at 11 33 26 AM" src="https://user-images.githubusercontent.com/14263346/114003978-81fb2600-985e-11eb-8866-54c7696694e9.png">

Reply:
Apologies, fixed